provider/source/src/currency/extended.rs

```diff
@@ -29,40 +31,34 @@ impl DataProvider<CurrencyExtendedDataV1Marker> for crate::SourceDataProvider {
             .get(req.id.marker_attributes.as_str())
             .ok_or(DataError::custom("No currency associated with the aux key"))?;
 
+        let display_name = currency.display_name.clone().ok_or(
```

Comment:
issue: this fell back to `one` and `other` before, which is necessary for the 11 entries that I posted

Reply:
ack